lib/rubocop/cop/lint/block_alignment.rb

```diff
@@ -148,12 +148,18 @@ def block_is_on_next_line?(begin_node, block_node)
           begin_node.loc.line != block_node.loc.line
         end
 
+        def ancestor_on_same_line(node)
+          node.ancestors.reverse.find do |ancestor|
+            ancestor.loc.line == node.loc.line
+          end
+        end
+
         def autocorrect(node)
-          key = node.children.first
+          ancestor_node = ancestor_on_same_line(node)
```

Comment:
I'd be happy to! :smile: Code should read like a newspaper article, meaning that you get the broad picture of what's going on at the top of the file, and the further down you get the more detailed it gets. So when you call `ancestor_on_same_line`, the definition of it should come after the call (so after `autocorrect`).